rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java

```diff
@@ -662,14 +661,17 @@ private String getEscapedTableName(String table) {
         if (tableNames != null && dbUnitConfig.isCacheTableNames()) {
             return tableNames;
         }
-        List<String> tables = new ArrayList<String>();
+        List<String> tables = new ArrayList<>();
         try (ResultSet result = getTablesFromMetadata(con)) {
             while (result.next()) {
                 String schema = resolveSchema(result);
                 if (!isSystemSchema(schema)) {
                     String name = result.getString("TABLE_NAME");
                     if (RESERVED_TABLE_NAMES.contains(name.toLowerCase())) {
                         name = escapeTableName(name);
+                    } else {
+                        // table name escaping may have been defined as well
+                        name = getEscapedTableName(name);
```

Comment:
Can you rename `getEscapeTableName` to `applyDBUnitEscapePattern`?

Thank you!

Reply:
I just renamed on your branch, thanks for the contribution!